src/parser/grammar.lalrpop

```diff
@@ -396,7 +396,19 @@ RecordField: (FieldPath, RichTerm) = {
         };
 
         (path, term)
-    }
+    },
+     <l: @L> <t: !> <r: @R> => {
+        let pos = mk_pos(src_id, l, r);
+        errors.push(t.clone());
+
+        let expr = RichTerm::new(Term::ParseError(
+            crate::error::ParseError::from_lalrpop(t.error, src_id)), pos);
+
+        let elem = FieldPathElem::Expr(expr);
+        let path = vec![elem];
+        let term = RichTerm::new(Term::MetaValue(MetaValue::new()), pos);
+        (path, term)
+    },
```

Comment:
I think this code has been copy paster from another rule, isn't it? It might be interesting to factor this out in a separate rule:

```
Error : RichTerm =  <l: @L> <t: !> <r: @R> => {
        let pos = mk_pos(src_id, l, r);
        errors.push(t.clone());

        RichTerm::new(Term::ParseError(
            crate::error::ParseError::from_lalrpop(t.error, src_id)), pos)
};
```

And then use this instead:
```suggestion
     <Error> => {
        let path = vec![FieldPathElem::Expr(<>)];
        let term = RichTerm::new(Term::MetaValue(MetaValue::new()), pos);
        (path, term)
    },
```

And use error as well in the Uniterm Rule